src/renderer/components/message/Message.tsx

```diff
@@ -26,7 +33,7 @@ const Avatar = (contact, onContactClick) => {
       ? String.fromCodePoint(codepoint).toUpperCase()
       : '#'
     return (
-      <div className='author-avatar default' alt={alt} onClick={onClick}>
+      <div className='author-avatar default' aria-label={alt} onClick={onClick}>
```

Comment:
Shouldn't we have alt + aria-label? I guess the idea of the alt war that you can hover over it and see the name. Maybe not needed here ux wise?

Reply:
alt is not a valid property on div (in react), so we should probably remove the thing here.